src/core/components/dht.js

```diff
@@ -164,22 +155,26 @@ module.exports = (self) => {
      * Find the closest peers to a given `PeerId`, by querying the DHT.
```

Comment:
Just above here we have `// TODO: Implement recursive providing` this needs to callback with an error.